views/js/qtiCreator/helper/saveChanges.js

```diff
@@ -18,7 +18,9 @@
 /**
  * @author Jean-Sébastien Conan <jean-sebastien@taotesting.com>
  */
-define(function () {
+define([
+    'jquery'
```

Comment:
may i ask why ?

Reply:
because jquery is used for trigger action below